.github/pull_request_template.md

```diff
@@ -1,10 +1,22 @@
-**Related Issue(s)**:  ...
+### Proposed Changes
+ <!--- In case of a bug: Describe what caused the issue and how you solved it-->
 
-**Proposed changes**:
-- ...
+### Related Issues
+- fixes #issue-number
```

Comment:
Very personal opinion: I like issues to be the first thing contributors read when opening a PR. It makes them think that they should open one before proceeding :grin: